src/uct/api/uct.h

```diff
@@ -221,6 +234,10 @@ struct uct_iface_attr {
 
     size_t                   iface_addr_len; /**< Size of interface address */
     size_t                   ep_addr_len;    /**< Size of endpoint address */
+
+    double                   overhead;     /**< Message overhead, seconds */
```

Comment:
it should be the overhead of every message, which limits the message rate and also added to latency